test/helpers.js

```diff
@@ -56,9 +56,33 @@ testHelpers.validateJson = function(json) {
   assert.ok(json.links instanceof Object, "Response should have a links block");
   assert.ok(!(json.errors instanceof Object), "Response should not have any errors");
   assert.equal(typeof json.links.self, "string", "Response should have a \"self\" link");
+  testHelpers.validatePagination(json);
   return json;
 };
 
+testHelpers.validatePagination = function(json) {
+  if (!json.meta.page) return;
+  if (!(json.data instanceof Array)) return;
+
+  var page = json.meta.page;
+  var expectedCount = null;
+  if ((page.offset + page.limit) > page.total) {
```

Comment:
I personally wouldn't bother with the redundant `()`. Do you reckon it deters from readability?